tests/test_platforms_dummy.py

```diff
@@ -133,15 +145,15 @@ def test_dummy_double_sweep(parameter1, parameter2, average, acquisition, nshots
 
     if average:
         results_len = (
-            len(results[pulse.qubit].magnitude)
+            len(results[pulse.qubit].magnitude.flatten())
```

Comment:
In #499 these tests were fixed to now check for the shape instead of `len`. If you merge the latest main to this branch and fix the conflicts you should get this change.